discord-services.js

```diff
@@ -92,7 +90,7 @@ module.exports.channelIDs = {
     /**
      * The admin console where admins can run commands.
      */
-    adminConsolChannel : '748955441484005488',
+    adminConsolChannel : '803242206814011433',
```

Comment:
I noticed that throughout the repo we sometimes call it adminConsoleChannel (which means it doesn't work) and sometimes call it adminConsolChannel and  so we should standardize it?

Reply:
yes will follow your convention of adminConsoleChannel